sdk/spring-boot-starter/src/main/scala/kalix/spring/boot/KalixContainerInitializer.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.spring.boot
+
+import java.io.File
+
+import scala.concurrent.Future
+
+import com.typesafe.config.Config
+import com.typesafe.config.ConfigFactory
+import kalix.devmode.KalixProxyContainer
+import org.slf4j.LoggerFactory
+import org.springframework.context.ApplicationContextInitializer
+import org.springframework.context.ApplicationListener
+import org.springframework.context.ConfigurableApplicationContext
+import org.springframework.context.event.ContextClosedEvent
+
+class KalixContainerInitializer
```

Comment:
For now, only integration with Spring. 

Later, we want to have a similar initializer for Java/Protobuf and Scala/Protobuf. Not yet clear to me the best way to do that though.